CONTRIBUTING.md

```diff
@@ -132,7 +132,23 @@ Note: Tycho always allows references to locally built artifacts, even if they ar
 
 ### Updating the Equinox and JDT dependencies of Tycho
 
-Tycho has Maven dependencies to Equinox and JDT, so these artifact are used from  Maven  Central repository. 
+Tycho has Maven dependencies to Equinox and JDT, so these artifact are used from  Maven  Central repository.
+
+### Profiling the Tycho build
+
+To understand where the build spends most of its time, you can try the following approaches:
+
+#### Add timestamps to Maven logging
+
+You can [add a timestamp to each log line](https://blogs.itemis.com/en/in-a-nutshell-adding-timestamps-to-maven-log-output) produced by Maven. This is the most easy to apply, but you have to do the calculation of the runtime of different goals yourself.
+
+#### Add Maven profiler
+
+Download the [Maven profiler extension](https://github.com/jcgay/maven-profiler) and add it to your aggregator project. It will produce an HTML report for each goal.
+
+#### Yourkit YouMonitor
+
+[Yourkit YouMonitor](https://www.yourkit.com/youmonitor/) (not to be confused with Yourkit Profiler) can be used to measure the build time steps. It reports the timing for Maven mojos, Ant goals etc. You need to register it as a Java agent for your build. It allows easy comparison of multiple builds, therefore it's really nice for trying different optimizations and configurations. Be aware the free license is only available for local builds, not for CI servers.
```

Comment:
Can you add an example how one can run a tycho build locally with the free license?